feature: enhance CORS support

rmikalkenas opened the pull request.

rmikalkenas committed:
feature: enhance CORS support

rustatian commented:
Hey @rmikalkenas 👋🏻 
Awesome, thanks for the PR 👍🏻 
I also added a new option called `debug` to debug CORS requests/responses. This is separate option since I can't pass RR logger to the CORS library: https://github.com/roadrunner-server/headers/blob/master/config.go#L32
Could you please add it as well?
I also slightly updated your merged PR, you may have a look at the changes here: https://github.com/roadrunner-server/headers/commit/dc73b7adf25529490ea390ff1fb2bbbad8cbd4f1

rmikalkenas commented:
> Hey @rmikalkenas 👋🏻 Awesome, thanks for the PR 👍🏻 I also added a new option called `debug` to debug CORS requests/responses. This is separate option since I can't pass RR logger to the CORS library: https://github.com/roadrunner-server/headers/blob/master/config.go#L32 Could you please add it as well? I also slightly updated your merged PR, you may have a look at the changes here: [roadrunner-server/headers@dc73b7a](https://github.com/roadrunner-server/headers/commit/dc73b7adf25529490ea390ff1fb2bbbad8cbd4f1)

Great addition :tada: updated the PR :+1:

rustatian reviewed (approved):
LGTM, thank you @rmikalkenas 👍🏻
